tests/test_bin_rucio.py

```diff
@@ -2175,7 +2175,7 @@ def test_update_rule_no_child_selfassign(self):
         cmd = r"rucio list-rules {0}:{1} | grep {0}:{1} | cut -f1 -d\ ".\
```

Comment:
I was debating this as well. I left them as is for the moment because I wanted to ensure the old commands still worked as expected (with the release schedule requiring them to still be functional). I'd like to move them to the new format soon though.